### ISSUE TITLE
Pictures, how they should be uploaded?

### ISSUE THREAD

**Issue by vladosmolovskiy** [allegro/allegro-api#8674]
Can I use links from s3 let's say or google drive? Or which way pictures should be uploaded on product creation API?

**Comment by Lukasz-Zurek** [comment]
No, you should provide direct links to the images. You will find more information [here](https://developer.allegro.pl/tutorials/jak-jednym-requestem-wystawic-oferte-powiazana-z-produktem-D7Kj9gw4xFA#wlasne-zdjecia-i-opis-oferty).
